utils/outputwriter/outputwriter.go

```diff
@@ -112,6 +114,7 @@ type MarkdownOutput struct {
 	avoidExtraMessages      bool
 	showCaColumn            bool
 	entitledForJas          bool
+	connectedToFrogbotRepo  bool
```

Comment:
I'm not sure that connectedToFrogbotRepo is the best name.
Can we use "hasInternetConnection" instead?